editor/scene_create_dialog.cpp

```diff
@@ -131,6 +132,18 @@ void SceneCreateDialog::update_dialog() {
 	root_name = root_name_edit->get_text().strip_edges();
 	if (root_name.is_empty()) {
 		root_name = scene_name.get_file().get_basename();
+
+		if (root_name.is_empty())
+		{
```

Comment:
Please bring the code in line with [style guidelines](https://docs.godotengine.org/en/stable/contributing/development/code_style_guidelines.html). You can use `clang-format` to auto-format.